docs/setup_help.md

```diff
@@ -0,0 +1,66 @@
+---
+id: setup_help
+title: Set Up Help
```

Comment:
Maybe Setup Details?
```suggestion
title: Setup Details
```

Reply:
Great suggestion! was trying to find a better name